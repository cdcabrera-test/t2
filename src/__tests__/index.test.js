const { t2, opts } = require('../');

describe('t2', () => {
  it('should have specific defined methods and properties', () => {
    expect(t2).toBeDefined();
    expect(opts).toBeDefined();
  });

  it('should export an object', () => {
    expect(t2('testing')).toMatchSnapshot('output');
  });
});
