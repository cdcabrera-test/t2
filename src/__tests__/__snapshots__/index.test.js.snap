// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`t2 should export an object: output 1`] = `
Object {
  "str": "testing",
}
`;
